src/components/Button/button.scss

```diff
@@ -0,0 +1,33 @@
+@import './variables';
+
+
+.btn {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 25px 80px;
+    background: $btn-color;
+    box-shadow: $box-shadow-btn-cateogry;
+    border-radius: 10px;
+    margin: 45px 0px;       
```

Comment:
Margin shouldn't be here, it should be handled in the component where we would be using this button

Reply:
Done